PAGE_OVERRIDES.md

```diff
@@ -0,0 +1,169 @@
+# Overrides
+
+💡 These notes cover the development part of the content overrides. For usage check [delta-config](https://github.com/NASA-IMPACT/delta-config/).
```

Comment:
A link to specific part of the doc might be more helpful. https://github.com/NASA-IMPACT/delta-config/blob/feature/component-overrides/docs/CONFIGURATION.md#component-overrides

Also, this is a crucial part of setting up overrides content, I think we can use a stronger words that you need to read this doc first.

Reply:
Agree with the link. Will update once merged so we have the correct url.

These docs are to be used only by the developers to create the overrides. I don't think it should be needed to read the other ones. This "developer docs" file should contains all that is needed to understand what they are and how to create them for the end user. These end users don't need to know how they're created, but only how to use them. What do you feel is missing from these docs?

Reply:
but developers (or anybody who wants to override) still need to start from defining overrides in delta.config? Let's say somebody decides to override about page.  I think it is unlikely that an editor is going to set up `delta.config`, then ask a developer to write a `ComonentOverride`.  It would be helpful for a developer to be able to see a whole workflow.

Reply:
I see what you mean. As far as I can understand, in creator scenario, a developer still needs to edit `mock/delta.config.js` to test with what I am writing, right? I think this step is missed from this doc.

Reply:
yes. that is correct. I'll look into adding a little info on that

Reply:
@hanbyul-here Added a piece around testing